lib/racecar/liveness_probe.rb

```diff
@@ -0,0 +1,73 @@
+require "fileutils"
+
+module Racecar
+  class LivenessProbe
+    def initialize(message_bus, file_path, max_interval)
+      @message_bus = message_bus
+      @file_path = file_path
+      @max_interval = max_interval
+      @subscribers = []
+    end
+
+    attr_reader :message_bus, :file_path, :max_interval, :subscribers
+    private     :message_bus, :file_path, :max_interval, :subscribers
+
+    def check_liveness_within_interval!
+      liveness_event_within_interval? or
+        raise "Racecar healthcheck failed: No liveness within interval #{max_interval}s. Last liveness at #{last_liveness_event_at}, #{elapsed_since_liveness_event} seconds ago."
```

Comment:
Could we just write to stderr and `exit(1)` here? Otherwise we'll clobber the logs with stack traces.

Reply:
Yes, that's a good call.

Generally I prefer to raise, but the stack trace doesn't help.